draft-kelly-json-hal.txt

```diff
@@ -420,26 +420,26 @@ Internet-Draft     JSON Hypertext Application Language          May 2016
    meaning and/or behaviour of the target Resource.  This will improve
    the discoverability of the API.
 
-   The CURIE Syntax [W3C.NOTE-curie-20101216] MAY be used for brevity
-   for these URIs.  CURIEs are established within a HAL document via a
-   set of Link Objects with the relation type "curies" on the root
+8.2.1.  CURIEs
+
+   Private-use scheme names (as defined by [RFC7595]) MAY be used.
+   Generally, this is done for brevity of these URIs.  To support
```

Comment:
I think we should just refer to them as "simple URIs" (ie. ones that are just `scheme:path`)